src/utils.py

```diff
@@ -1267,16 +1268,34 @@ def update_config(args: Dict[str, Any], default_seed: int = 123) -> None:
             # pass in a value and this key is already in the
             # configuration dict, add the current value to args.
             args[k] = getattr(CFG, k)
-    # Maintain the invariant that CFG has some seed and some
-    # experiment_id set. This is very useful in unit tests, where
-    # there are often no command line args being passed.
-    args["seed"] = args.get("seed", default_seed)
-    args["experiment_id"] = args.get("experiment_id", "")
```

Comment:
Where is the default setting of experiment_id happening now?

Reply:
it defaults to an empty str in args.py